app/components/machine/driver-amazonec2/template.hbs

```diff
@@ -96,13 +96,20 @@
               {{#if choice.isVpc}}
                 <hr class="mt-20 mb-20" />
                 <div class="radio">
-                  <label class="acc-label pb-0">{{radio-button selection=selectedSubnet value=choice.value}} {{choice.label}}</label>
+                  <label class="acc-label pb-0">{{radio-button selection=selectedSubnet value=choice.value}} {{choice.label}}
+                    {{#if choice.tags.length}}
+                      <i class="icon icon-tag"></i> {{choice.tags}}
```

Comment:
Can you wrap these in <span class="tag bg-info">...</span> so they stand out

Reply:
Thanks Vince. Updated.
Do you mean something like this ?

Reply:
Sorry GitHub ate the tag... `<span class="tag bg-info">...</span>`

Reply:
Thanks. Please let me know if I need to change something else